spec/features/games/show_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe "As a user" do
+  it "i can see a game show page" do
+    game1 = create(:game)
+    visit "/games/#{game1.id}"
+
+    expect(page).to have_content("Title of game")
+    expect(page).to have_content("description of game")
+    expect(page).to have_content("5 Stars")
```

Comment:
What rating should we use? stars is functional, but it could be spooky themed

Reply:
Maybe this is JS magic but could we do a meter?